### DISCUSSION TITLE
Edge Functions : Breaking Changes on August 1

### DISCUSSION THREAD

**Discussion by ijaspreetsinghh** [supabase/supabase#6745] [Questions] [answered]
Hi, we all know the edge function will have a breaking change on August 1, 2022, but I strongly need to use these functions, but worried about future breakage.
Just wanna know if I implement a function, will it crash in future (on August 1, 2022 ) or changes will only be applicable for future functions like deployed after August 1, 2022.
![image](https://user-images.githubusercontent.com/44422128/167410373-e317e596-30f9-4af0-afbf-6a20e0e33937.png)

**Comment by GaryAustin1** [comment, marked as answer]
I don't have inside information, but I DO NOT believe that means there will be a change on 8/1.  I believe it means there could be breaking changes up until that point that may or may not affect a current function.  At least that was the sense conveyed in a video announcement I saw.
